commands/Fun/cat.js

```diff
@@ -0,0 +1,22 @@
+const snek = require("snekfetch");
+
+exports.run = async (client, msg) => {
+  const { body } = await snek.get("http://random.cat/meow");
+  await msg.send({ files: [{ attachment: body.file, name: `cat.${body.file.split(".")[2]}` }] }).catch(e => msg.channel.send(e));
```

Comment:
Do the same changes as requested in [klasa-pieces#12](https://github.com/dirigeants/klasa-pieces/pull/12) and this PR will be fine to merge.